helm/ingress-nginx/values.yaml

```diff
@@ -17,18 +17,18 @@ commonLabels: {}
 
 controller:
   name: controller
-  enableAnnotationValidations: true
+  enableAnnotationValidations: false
   image:
     ## Keep false as default for now!
     chroot: false
-    # registry: registry.k8s.io
-    image: giantswarm/ingress-nginx-controller
+    registry: registry.k8s.io
+    image: ingress-nginx/controller
     ## for backwards compatibility consider setting the full image url via the repository value below
     ## use *either* current default registry/image or repository format or installing chart by providing the values.yaml will fail
     ## repository:
-    tag: "v1.9.6"
-    digest: ""
-    digestChroot: ""
+    tag: "v1.10.0"
+    digest: sha256:42b3f0e5d0846876b1791cd3afeb5f1cbbe4259d6f35651dcc1b5c980925379c
+    digestChroot: sha256:7eb46ff733429e0e46892903c7394aff149ac6d284d92b3946f3baf7ff26a096
```

Comment:
We can not go to v1.10.0 as there is no `chroot` image, yet. Also the digests need to be omitted.